stories/components/LessonCard.stories.tsx

```diff
@@ -6,6 +6,13 @@ export default {
   title: 'Components/LessonCard'
 }
 
+export const Dummy: React.FC = () => <div>Dummy Lesson Card</div>
```

Comment:
I didn't dig too much into the dependencies of LessonCard, but have you explored trying to mock `useQuery`?

Reply:
At least for the time being? I understand the end goal is to decouple them, but for now, we might as well try to mock the dependencies

Reply:
After unraveling the mystery of the `shouldNotGetCount` prop, I wasn't feeling particularly motivated to do more work to add a space between the two words on the button.

But you make a good point here. I have a working mock implementation. I'll add it to the PR in the next commit.

There's at least one rough edge that I'm aware of. The mock data is a little distracting in the story file. I feel like it should be hidden somewhere in a distant sub-directory.

Please take another look after the next commit and let me know what you think.